lookout/style/typos/cmdline_tools.py

```diff
@@ -174,6 +174,14 @@ def add_parser(name, help):
         help="Directory where to download repositories from the dataset. It is strongly \
               recommended to set this parameter if there are more than 20 repositories \
               in the dataset. Temporary directory is used if not set.")
+    typo_commits_report_parser.add_argument(
+        "--checkpoint-dir", default=None,
+        help="Directory where to save intermediate reports to save the progress. "
```

Comment:
```suggestion
        help="Directory where to save the intermediate reports, so that we do not have to start from scratch in case of an error. "
```